libs/design-system/src/lib/Components/NetworkDetailsCard/index.tsx

```diff
@@ -56,11 +58,13 @@ const NetworkDetailsCard = ({
       </div>
 
       <div className="px-8 py-4">
-        <p className="mb-2 text-xs uppercase text-primary-gray-200">RSS Feed</p>
+        <p className="mb-2 text-xs uppercase dark:text-primary-blue-dark text-primary-gray-200">
+          RSS Feed
+        </p>
         {/* @ts-ignore */}
         <Link
           href={rssFeed}
-          className="flex items-center text-sm text-primary-blue hover:cursor-pointer dark:text-primary-gray-50"
+          className="flex items-center text-sm hover:cursor-pointer"
```

Comment:
```suggestion
          className="flex items-center text-sm"
```

Links are still a bit of a work in progress. Non-external links require passing the <a href... tag as part of its children. I think the external ones will be updated to work the same way. cc @10thfloor